lib/macos.c

```diff
@@ -30,7 +30,14 @@
 
 #include "macos.h"
 
+#if defined(__clang__)
 #include <SystemConfiguration/SCDynamicStoreCopySpecific.h>
+#else
+#include <CoreFoundation/CFDictionary.h>
+typedef const struct CF_BRIDGED_TYPE(id) __SCDynamicStore * SCDynamicStoreRef;
+CFDictionaryRef __nullable
+SCDynamicStoreCopyProxies(SCDynamicStoreRef __nullable store);
+#endif
```

Comment:
If you decide to go with this, I think a clarifying comment about why this is done could be valuable.